README.md

```diff
@@ -4,40 +4,55 @@
 
 # Spine Event Engine Base
 
-This repository contains the code of foundation of the Spine Event Engine framework which includes:
-
-* The framework [base](base) data types and utilities.
-* [Utilities for testing](testlib).
-* Gradle plug-ins:
-  * [Spine Model Compiler](tools/model-compiler) — transforms a domain model defined in proto files 
-    into the Java code.
-  * [Spine Javadoc Prettifier](tools/javadoc-prettifier) — processes Javadocs of generated files.
-* [Protoc plugin](tools/protoc-plugin) — a plug-in for Google Protobuf compiler for generating 
-  custom code for framework-specific message types.
-* [Proto JS plugin](tools/proto-js-plugin) — a plug-in that assists Protobuf JS compiler in 
-  JavaScript code generation.
-* [Spine Javadoc Filter](tools/javadoc-filter) — excludes elements annotated with 
+This repository contains the foundation of the Spine framework which includes the following modules:
+
+* **[`base`](base)** — the framework base data types and utilities for working with them.
+* **[`testlib`](testlib)** - utilities for testing.
+
+* **[`tools/javadoc-style`](tools/javadoc-style)** — a Gradle plugin which processes Javadocs of
+  generated Java files to make them look less ugly.
+* **[`tools/javadoc-filter`](tools/javadoc-filter)** — excludes elements annotated with
   `io.spine.Internal` from the generated doc.
-* [Spine Error Prone Checks](tools/errorprone-checks) — the custom Error Prone checks for the
-  Spine projects.
+  
+  
+* **Model Compiler for Java** which consists of the following sub-modules:
+  * **[`tools/mc-java`](tools/mc-java)** — a Gradle plugin responsible for enabling Java
+    code generation.
+  * **[`tools/mc-java-protoc`](tools/mc-java-protoc)** — plugs-into Google Protobuf compiler for
+    generating framework-specific Java code.
+  * **[`tools/mc-java-protoc`](tools/mc-java-protoc)** — static code analyzers for Spine-based
```

Comment:
Please fix the reference to the module. It's duplicated in the previous list item.